pipeline/metering/shadow_evidence.py

```
@@ -0,0 +1,201 @@
+            rec = json.loads(ln)
+        except json.JSONDecodeError as e:
+            errs.append(f"{os.path.basename(path)} 第 {i} 行 JSON 畸形: {e}")
+            break
+        if rec.get("seq") != i:
+            errs.append(f"{path} 第 {i} 行 seq={rec.get('seq')} 断号")
+        if rec.get("prev_hash") != prev:
+            errs.append(f"{path} 第 {i} 行 prev_hash 断链")
+        if rec.get("hash") != content_hash(rec):
+            errs.append(f"{path} 第 {i} 行 hash 重算不符（篡改）")
+        if not str((rec.get("subject") or {}).get("tenant") or "").strip():
+            errs.append(f"{path} 第 {i} 行 tenant 缺失（AC-4c）")
```

Ruff (0.16.2)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)